p2p_wallet/Scenes/Main/Send/Input/SendInputViewModel.swift

```diff
@@ -70,6 +70,7 @@ final class SendInputViewModel: BaseViewModel, ObservableObject {
     private let source: SendSource
     private var wasMaxWarningToastShown: Bool = false
     private let preChosenAmount: Double?
+    private let sendViaLinkSeed: String?
```

Comment:
No need to save sendViaLinkSeed to detect sendViaLink, you can always check via `currentState.isSendingViaLink`